src/sfctl/custom_deployment.py

```diff
@@ -0,0 +1,131 @@
+    print("Creating resource: ", resource_name, "of type: ", resource_type.name)
+    if resource_type == ResourceType.application:
+        application_description = load_json(resource)
+        client.mesh_application.create_or_update(resource_name, application_description.get('description')) # pylint: disable=line-too-long
+    elif resource_type == ResourceType.volume:
+        volume_description = load_json(resource)
+        client.mesh_volume.create_or_update(resource_name, volume_description.get('description')) # pylint: disable=line-too-long
+    elif resource_type == ResourceType.network:
+        network_description = load_json(resource)
+        client.mesh_network.create_or_update(resource_name, network_description.get('description').get('name'), network_description.get('description').get('properties')) # pylint: disable=line-too-long
+    elif resource_type == ResourceType.secret:
+        secret_description = load_json(resource)
+        client.mesh_secret.create_or_update(resource_name, secret_description.get('description').get('properties'), secret_description.get('description').get('name')) # pylint: disable=line-too-long
+    elif resource_type == ResourceType.secretValue:
+        secret_value_description = load_json(resource)
+        fully_qualified_resource_name = secret_value_description.get('fullyQualifiedResourceName').split('/') # pylint: disable=line-too-long
+        secret_value_resource_name = fully_qualified_resource_name[1]
+        client.mesh_secret_value.add_value(resource_name, secret_value_resource_name, secret_value_description.get('description').get('name'), secret_value_description.get('description').get('properties').get('value')) # pylint: disable=line-too-long
+    elif resource_type == ResourceType.gateway:
+        gateway_description = load_json(resource)
+        client.mesh_gateway.create_or_update(resource_name, gateway_description.get('description')) # pylint: disable=line-too-long
+    else:
+        raise CLIError('Invalid resource type found %s' %(resource))
+
+def mesh_deploy(client, input_yaml_file_paths, parameters=None):
+    """ This function
+        1.Uses sfmergeutility to merge, convert and
+            order the resources.
+        2. Deploys the resources in the order suggested by the utility
+    """
+    file_path_list = []
+    if os.path.isdir(input_yaml_file_paths):
+        if not os.path.exists(input_yaml_file_paths):
+            raise CLIError("The specified directory %s does not exist or you do not have access to it" %(input_yaml_file_paths)) # pylint: disable=line-too-long
+        file_path_list = list_files_in_directory(input_yaml_file_paths, ".yaml")
+    else:
+        file_path_list = input_yaml_file_paths.split(',')
+        for file_path in file_path_list:
+            if not os.path.exists(file_path):
+                raise CLIError("The specified file %s does not exist or you do not have access to it" %(file_path)) # pylint: disable=line-too-long

```

Comment:
nit: %s -> "%s"

Reply:
took your feedback from and applied it to the code at it's newest location